pandas/core/indexes/base.py

```diff
@@ -1283,12 +1283,12 @@ def set_names(self, names, level=None, inplace=False):
         Parameters
         ----------
         names : str or sequence
-            name(s) to set
+            Name(s) to set.
         level : int, level name, or sequence of int/level names (default None)
```

Comment:
We try to have the types more than description in this line. Something like the next would be more standard:
`level: int, str, list of int or str, optional`

In the description you can expand on explaining that it is the level index or name.

https://pandas.pydata.org/pandas-docs/stable/contributing_docstring.html#section-3-parameters